app/controllers/submit.js

```diff
@@ -792,6 +780,19 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             model.set('isPublished', true);
             node.set('public', true);
 
+            if (this.get('applyLicense')) {
+                let newCopyrightHolders = [''];
+                if (this.get('basicsLicense.copyrightHolders') && this.get('basicsLicense.copyrightHolders').length) {
+                    newCopyrightHolders = this.get('basicsLicense.copyrightHolders').split(',');
+                }
+                if (node.get('nodeLicense.year') !== this.get('basicsLicense.year') || node.get('nodeLicense.copyrightHolders') !== newCopyrightHolders) {
```

Comment:
Question about a particular case for this line.  Say my original node has a copyrightHolders Dawn.  The preprint created on this node has no copyrightHolders, nor did the user ever enter any because the license that they selected didn't require it. applyLicense is true.  It seems strange to me that we would then erase the copyrightHolders on the node in this instance.